File: SampleSetup/Output/PHP/SampleProject/Protocol/Body/ApiUserRegisterArg.php
<?php
namespace SampleProject\Protocol\Body;

class ApiUserRegisterArg {
    /** @var string */ public $UserName;
    /** @var int */ public $UserNameKind;
    /** @var string */ public $Password;

    /** @var string|null */ public $FirstName;

    /** @var string */ public $LastName;
    /** @var bool */ public $IsGuest;
    /** @var string */ public $MobileMSISDN;

    public function __construct()
    {
        $this->UserName = "";
        $this->UserNameKind = 0;
        $this->Password = "";
        $this->FirstName = null;
        $this->LastName = "";
        $this->IsGuest = false;
        $this->MobileMSISDN = "";
    }
}

File: SampleSetup/Protocol/Body/ApiUserRegisterArg.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body
{
    public class ApiUserRegisterArg
    {
        public string UserName { get; set; } = "";

        public int UserNameKind { get; set; } = 0;

        public string Password { get; set; } = "";

        [Nullable]
        public string FirstName { get; set; } = null;

        public string LastName { get; set; } = "";

        public bool IsGuest { get; set; } = false;

        [Obsolete("Please use MobileMSISDN")]
        public string Mobile { get; set; } = "";

        public string MobileMSISDN { get; set; } = "";
    }
}
